test/test_prototype_transforms.py

```diff
@@ -1899,3 +1899,22 @@ def test_transpose_dimensions(dims):
             assert type(transformed_value) == torch.Tensor
         else:
             assert transformed_value is value
+
+
+class TestUniformTemporalSubsample:
```

Comment:
We currently don't have automated tests for testing the transforms that wraps a kernel. Thus, I don't have any objections here. I'm going to look into more structured tests for this as soon as the we pass the tight deadlines.